2023-07-29 13:06:02.245 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-29 13:06:02.246 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration snooz which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-29 13:06:02.246 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration dreo which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2023-07-29 13:06:04.936 ERROR (MainThread) [custom_components.hacs] HACS is disabled - invalid_token
2023-07-29 13:06:04.939 ERROR (MainThread) [custom_components.hacs] Bad credentials
Traceback (most recent call last):
  File "/config/custom_components/hacs/base.py", line 464, in async_can_update
    response = await self.async_github_api_method(self.githubapi.rate_limit)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/hacs/base.py", line 522, in async_github_api_method
    raise HacsException(_exception)
custom_components.hacs.exceptions.HacsException: Bad credentials
2023-07-29 13:06:06.688 WARNING (MainThread) [homeassistant.config_entries] Config entry 'Franâ€™s Roku Streambar' for roku integration not ready yet: Invalid response from API: Error occurred while communicating with device; Retrying in background
2023-07-29 13:31:07.778 WARNING (MainThread) [zigpy_deconz.api] No response to 'Command.aps_data_indication' command with seq id '0x4c'
2023-07-29 13:59:40.020 ERROR (MainThread) [pyhap.hap_handler] ('192.168.0.155', 49204): Client 26a4b13d-b854-4f72-89d3-75c870c2321a with uuid {UUID('6723c492-82a4-4b26-9a6a-4df07bd0f86d'): b"\xe19v\xfcTe\x82v\xe4o\xaaTS\xb2'z3\x82\xcc5T\xb2m\x0c\xa2\xc7\x97*\xaa\xf9\x89\xce", UUID('4372d08d-dbf3-4366-b0d0-35f76c5ba2b9'): b'\xb4\xb7\xd8\x9b\x16n\x9c\xe1\xa6i\xa9\xec_m\x9f\x00\x90jNH\xfd\xd8\xfa\xc5i\xbb\x17h\x11A\xa5e'} attempted pair verify without being paired first (paired clients=HASS Bridge).
